src/webchat-ui/components/plugins/MessagePluginRenderer.tsx

```diff
@@ -80,10 +85,10 @@ export default ({ message, config, onSendMessage, plugins, isFullscreen, onSetFu
                     <MessageRow
                         key={key}
                         align={message.source === 'bot' ? 'left' : 'right'}
-                        className={message.source === 'bot' ? 'webchat-chat-message-bot': 'webchat-chat-message-user'}
+                        className={message.source === 'bot' ? regularMessageBot : regularMessageUser }
```

Comment:
this is not part of the message plugin yet, should be `webchat-message-row` +  `user` or `bot`

Reply:
fixed